cdci_data_analysis/analysis/instrument.py

```diff
@@ -93,6 +93,7 @@ def __init__(self,
                  instrumet_query,
                  input_product_query=None,
                  asynch=True,
+                 development=False,
```

Comment:
As far as the behaviour can be more general, not only checking for 'development' role, this should probably be renamed to smth like "restricted_access"

Reply:
yeah makes sense